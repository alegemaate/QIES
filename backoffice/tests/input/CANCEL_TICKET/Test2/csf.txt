11111 1000 2 Guicci 20181116