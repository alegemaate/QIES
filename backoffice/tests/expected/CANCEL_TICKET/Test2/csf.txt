11111 1000 1 Guicci 20181116